packages/@okta/vuepress-site/docs/guides/device-context/main/nodeexpress/integrationsteps.md

```diff
@@ -0,0 +1,15 @@
+Add the following code to your app to send the `X-Device-Token` header with a device ID attached.
+
```

Comment:
Add the following code to your app to set the `X-Device-Token`with a device ID.

Reply:
fixed